### PR TITLE
package.json Cleanup

### PR DESCRIPTION
**Pull request by swashcap**
I noticed a few fields were missing or incorrect in _package.json_ (see comments).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,13 +20,16 @@
   "main": "index.js",
   "module": "index.m.js",
   "peerDependencies": {
-    "preact": "*",
-    "redux-bundler": "*"
```
Comment (swashcap) [inline package.json:23]: `redux-bundler` isn't ever used in the source.

File: package.json [hunk 2]
```diff
@@ -19,13 +20,16 @@
...
+    "preact": "3.0.0 - 8.x.x"
```
Comment (swashcap) [inline package.json:23]: This package breaks for `preact@^10` so consumers know what's up.

File: package.json [hunk 3]
```diff
@@ -11,6 +11,7 @@
     "tap-spec": "4.1.1",
     "tape": "4.8.0"
   },
+  "homepage": "https://github.com/HenrikJoreteg/redux-bundler-preact",
```
Comment (swashcap) [inline package.json:14]: Maybe it's me, but I'm always linking to packages' source from npmjs.com. They consume this field to populate that link.

**Comment by HenrikJoreteg** [comment]
This looks great 👍

**Review by HenrikJoreteg — APPROVED** [review]

**Pull request merged**